package/build_appimage.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ ! -f ../bin/samrewritten ]; then
+    echo "Please build SamRewritten first."
```

Comment:
```suggestion
    echo "Building SamRewritten first..."
    cd $(dirname $(realpath $0))/..
    make
fi
```

Perhaps build automatically if it isn't already?